dockerfiles/init/modules/codenvy/templates/codenvy.env.erb

```diff
@@ -1,3 +1,11 @@
+# Include all custom CODENVY_* properties that user may has defined in its codenvy.env file
+<% ENV.each do |k,v| -%>
+<% if k.include? "CODENVY_" then  -%>
```

Comment:
Yes, it does as well.  The SAAS CLI packaging inherits directly from the Codenvy CLI packaging.  In fact, the `/init` portion of SAAS is a (mostly) perfect duplicate of the same in Codenvy.